repo/manifest/manifest_entry.go

```diff
@@ -5,8 +5,8 @@ import "time"
 // EntryMetadata contains metadata about manifest item. Each manifest item has one or more labels
 // Including required "type" label.
 type EntryMetadata struct {
-	ID      ID
-	Length  int
-	Labels  map[string]string
-	ModTime time.Time
+	ID      ID                `json:"id"`
```

Comment:
Does this break existing (already serialized) manifests?

Reply:
no, this struct was never serialized, with this change we'll start sending it over the wire.